src/freenas/usr/local/bin/ixdiagnose

```diff
@@ -146,6 +146,10 @@ fi
 
 echo "** 96%: Compressing System Logs"
 tar -chf - -C /var --exclude=log/journal --exclude=log/faillog --exclude=log/lastlog --exclude=log/sysstat log | tar -C $dir -xf -
+if [ -f "/tmp/failsafe_middlewared.log" ]; then
+	cp "/tmp/failsafe_middlewared.log" "$dir"
```

Comment:
If we have 5 old rotated files, might as well copy them here

Reply:
great point. fixed